### PR TITLE
test: consistent use of set_setting rather than adding to emcc_args. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -2017,7 +2016,8 @@ def test_memorygrowth_3(self):
       self.skipTest('test needs to modify memory growth')
 
     # checks handling of malloc failure properly
-    self.emcc_args += ['-s', 'ALLOW_MEMORY_GROWTH=0', '-s', 'ABORTING_MALLOC=0', '-s', 'SAFE_HEAP']
+    self.set_setting('ABORTING_MALLOC', 0)
```
Comment (aheejin) [inline tests/test_core.py:2019]:
```suggestion
    self.clear_setting('ABORTING_MALLOC')
    self.clear_setting('ALLOW_MEMORY_GROWTH')
```

**Review by aheejin — APPROVED** [review]

File: tests/test_core.py (hunk 1 above)
Comment (sbc100) [reply tests/test_core.py:2019]: ABORTING_MALLOC defaults to '1' so clearing is not enough, it needs to be explicitly set to 0

Comment (aheejin) [reply tests/test_core.py:2019]: Oh I see, I was miunderstanding what `clear_setting` meant.

**Comment by aheejin** [comment]
It seems this was merged not into `master` branch but into `minimal_runtime_closure` branch..? Is this intended?
By the way #13185, which uses `minimal_runtime_closure` branch, doesn't seem to contain these changes.

**Comment by sbc100** [comment]
I will reland this change on master once #13185 lands.

**Pull request merged**